noir-projects/aztec-nr/address-note/src/address_note.nr

```diff
@@ -44,12 +44,13 @@ impl NoteInterface<ADDRESS_NOTE_LEN, ADDRESS_NOTE_BYTES_LEN> for AddressNote {
     }
 
     // Broadcasts the note as an encrypted log on L1.
-    fn broadcast(self, context: &mut PrivateContext, slot: Field, ivpk_m: GrumpkinPoint) {
+    fn broadcast(self, context: &mut PrivateContext, slot: Field, ovpk_m: GrumpkinPoint, ivpk_m: GrumpkinPoint) {
```

Comment:
I swapped the order in the last few commits because I felt exactly the opposite. In the ERC20 interface you usually have order as from -> to and ovpk is a log for "from" and ivpk for "to".

I think it's the current naming that is messed up and has it inverted. But IDK